src/helpers/geoChart.ts

```diff
@@ -106,6 +106,13 @@ export function isDataOfReasonableSize(
     return locationData.length <= limit;
 }
 
+export function isLocationMissing(buckets: VisualizationObject.IBucket[]): boolean {
+    const locationBucket: VisualizationObject.IBucket = buckets.find(
+        (bucket: VisualizationObject.IBucket) => bucket.localIdentifier === LOCATION,
+    );
+    return !locationBucket || locationBucket.items.length === 0;
```

Comment:
it could be `!locationBucket.items.length`

Reply:
updated.
thanks :)